compression/compression-multiple-chunks.any.js

```diff
@@ -0,0 +1,63 @@
+// META: global=worker
+// META: script=pako/pako_inflate.min.js
+// META: timeout=long
+
+'use strict';
+
+// This test asserts that compressing multiple chunks should work.
+
+// ('Hello', 3) => TextEncoder().encode('HelloHelloHello')
+function makeExpectedChunk(input, numberOfChunk) {
+  let expectedChunk = '';
+  for (let i = 0; i < numberOfChunk; ++i) {
+    expectedChunk += input;
+  }
+  return new TextEncoder().encode(expectedChunk);
+}
+
+// ex) ('Hello', 3, 'deflate') => compress ['Hello', 'Hello', Hello']
+async function compressMultipleChunks(input, numberOfChunk, format) {
```

Comment:
Nitpick: should be "numberofChunks", not "numberOfChunk".

Reply:
Done.